nano/node/websocket.hpp

```diff
@@ -143,12 +143,20 @@ namespace websocket
 			return node;
 		}
 
+		/** Per topic subscription check */
+		bool any_subscription (nano::websocket::topic const & topic_a);
```

Comment:
Minor: `any_subscribers` or `has_subscribers` reads more naturally to me on call sites. WDYT?